openedx/core/djangoapps/content_libraries/api.py

```diff
@@ -815,6 +815,8 @@ def validate_can_add_block_to_library(
                     block_type=block_type, library_type=content_library.type,
                 )
             )
+    if block_type in settings.LIBRARY_UNSUPPORTED_BLOCKS:
+        raise IncompatibleTypesError(_('Libraries do not support this type of content yet.'))
```

Comment:
Should we include `block_type` in the message?

Reply:
I think the current message is enough as we are using the Toast to show the message.